client-report/index.html

```diff
@@ -1,87 +1,89 @@
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <style type="text/css">
+
+    /*begin http://bl.ocks.org/jensgrubert/7789216 */
+    .box {
+      font: 10px sans-serif;
+    }
+
+    .box line,
+    .box rect,
+    .box circle {
+      fill: steelblue;
       stroke: #000;
-      shape-rendering: crispEdges;
-      }
+      stroke-width: 1px;
+    }
+
+    .box .center {
+      stroke-dasharray: 3,3;
+    }
 
-      .x.axis path {
+    .box .outlier {
       fill: none;
       stroke: #000;
-      shape-rendering: crispEdges;
-      }
-      /*end  http://bl.ocks.org/jensgrubert/7789216 */
-
-      html, p, div, input, button {
-        font-family: "Helvetica Neue", "Helvetica", sans-serif;
-        font-weight: 400;
-        /*color: rgb(160, 160, 179);*/
-      }
-
-
-
-      @media print {
-          .move-left-20-for-print {
-            margin-inline-start: -20px;
-          }
-          .shrink-for-print-70 {
-            transform: scale(0.7);
-          }
-          .shrink-for-print-50 {
-            transform: scale(0.5);
-          }
-      }
-
-    </style>
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
-    <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon">
-    <link rel="icon" href="/favicon.ico" type="image/x-icon">
-  </head>
-
-  <body class="viewport">
-    <div id='root'>
-    </div>
-    <script src="https://d3js.org/d3.v4.min.js"></script>
-    <script src="https://d3js.org/d3-scale-chromatic.v1.min.js"></script>
-    <script src="/dist/report_bundle.js"></script>
-    <script>window.version = "NULL_VERSION";</script>
-
-  </body>
+    }
+
+    .axis {
+      font: 12px sans-serif;
+    }
+
+    .axis path,
+    .axis line {
+    fill: none;
+    stroke: #000;
+    shape-rendering: crispEdges;
+    }
+
+    .x.axis path {
+    fill: none;
+    stroke: #000;
+    shape-rendering: crispEdges;
+    }
+    /*end  http://bl.ocks.org/jensgrubert/7789216 */
+
+    html, p, div, input, button {
+      font-family: "Helvetica Neue", "Helvetica", sans-serif;
+      font-weight: 400;
+      /*color: rgb(160, 160, 179);*/
+    }
+
+
+
+    @media print {
+        .move-left-20-for-print {
+          margin-inline-start: -20px;
+        }
+        .shrink-for-print-70 {
+          transform: scale(0.7);
+        }
+        .shrink-for-print-50 {
+          transform: scale(0.5);
+        }
+    }
+
+  </style>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
+  <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon">
+  <link rel="icon" href="/favicon.ico" type="image/x-icon">
+</head>
+
+<body class="viewport">
+  <div id='root'>
+  </div>
+  <script src="https://d3js.org/d3.v4.min.js"></script>
+  <script src="https://d3js.org/d3-scale-chromatic.v1.min.js"></script>
+  <script src="/dist/report_bundle.js"></script>
```

Comment:
Regarding last comment, seems we already weren't doing the cache busting here, so maybe we're fine on that front, but possibly worth flagging whether we want to start doing that, and have webpack inject the script tag, as with participation (and I think admin) client(s). Traditionally, we've assumed that the report will be viewed by fewer people, having less load, and therefore less need for caching. But we're seeing an increasing trend of deployments that may be assuming open public access to reports, and should revisit these assumptions.

cc @xeeg

Reply:
Erm... nevermind. Looks like we were doing cache busting.

https://github.com/compdemocracy/polis/pull/1644/files#r1131615626